manager/integration/build-image.sh

```diff
@@ -18,6 +18,10 @@ case $(uname -m) in
 esac
 
 echo "Building for ${ARCH}"
+# update base image to get latest changes                                       
+BASE_IMAGE=`grep FROM package/Dockerfile  | awk '{print $2}'`                   
+docker pull ${BASE_IMAGE}
```

Comment:
Does it mean we will pull new base image for every builds?

Reply:
Yes, to avoid caching base image, and get new updated minor versions when available.